### PR TITLE
Fixing problem with appearing icons on the map and the graph

### PR DESCRIPTION
**Pull request by SzymonGorowski**
https://trello.com/c/cqWBGFc9/672-icon-appears-on-map-graph-bug

### CODE REVIEW

File: AirCasting/SessionViews/ExternalSessionsViews/ExternalSessionCard.swift [hunk 1]
```diff
@@ -57,6 +57,10 @@ struct ExternalSessionCard: View {
                 mapStatsDataSource?.dataSource.stream = newStream
                 uiStateHandler.changeSelectedStream(sessionUUID: session.uuid, newStream: newStream?.sensorName ?? "")
             })
+            .onChange(of: isMapButtonActive) { _ in
+                reorderButton.navigationLink(isActive: isMapButtonActive)
+                searchAndFollowButton.navigationLink(isActive: isMapButtonActive)
```
Comment (jankrzempek) [inline AirCasting/SessionViews/ExternalSessionsViews/ExternalSessionCard.swift:62]: Maybe we could change the naming here to sth like: `reorderButton.setVisability(to: isVisible)` by taking `isVisible in`? I am not sure about `isVisible` but nothing else came to my mind.

**Review by jankrzempek — APPROVED** [review]
Just naming a comment. Working cooool 🤩

File: AirCasting/MainTabBarView.swift [hunk 2]
```diff
@@ -207,11 +207,31 @@ class FinishAndSyncButtonTapped: ObservableObject {
 class ReorderButton: ObservableObject {
     @Published var reorderIsOn = false
     @Published var isHidden = false
+    
+    func isHidden(if isActive: Bool) {
```
Comment (pawelgil) [inline AirCasting/MainTabBarView.swift:211]: I'd call it `setHidden`

File: AirCasting/MainTabBarView.swift [hunk 3]
```diff
@@ -207,11 +207,31 @@ class FinishAndSyncButtonTapped: ObservableObject {
...
+            }
+        }
+    }
 }
 
 class SearchAndFollowButton: ObservableObject {
      @Published var searchIsOn = false
      @Published var isHidden = false
+    
+    func isHidden(if isActive: Bool) {
```
Comment (pawelgil) [inline AirCasting/MainTabBarView.swift:226]: I'd call it `setHidden`

**Review by pawelgil — APPROVED** [review]

File: AirCasting/MainTabBarView.swift [hunk 4]
```diff
@@ -207,11 +207,31 @@ class FinishAndSyncButtonTapped: ObservableObject {
 class ReorderButton: ObservableObject {
     @Published var reorderIsOn = false
     @Published var isHidden = false
+    
+    func setHidden(if isActive: Bool) {
+        if isActive {
```
Comment (anna1901) [inline AirCasting/MainTabBarView.swift:212]: What does it mean? What `isActive`?

**Pull request merged**